docs/maintenance/Team.mdx

```diff
@@ -0,0 +1,98 @@
+---
+id: team
+title: Team
+---
+
+import { TeamBioDivider } from '@site/src/components/team/TeamBioDivider';
+import { TeamBioList } from '@site/src/components/team/TeamBioList';
+
+The typescript-eslint organization is an independent open source project. Its
+members are volunteer open source developers dedicated to making the
+TypeScript linting experience great.
```

Comment:
Hmm, maybe?

```suggestion
The typescript-eslint organization is an independent open-source project. Its
members are volunteer open-source developers dedicated to making the
TypeScript linting experience great.
```


See: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source

Reply:
Heh, interesting nuance here. We use "open source" all but one existing places in the docs. I'm also not personally enamored with large corporate style guides - they can get archaic at times. Thanks though 😄